### DISCUSSION TITLE
vc-drawings下的polygon组件，鼠标在移动过程中不允许线段有交叉，这个要怎么扩展handleMouseMove事件呢？

### DISCUSSION THREAD

**Discussion by 1194964459** [zouyaoji/vue-cesium#526] [Q&A]
![polygon不允许有交叉](https://github.com/zouyaoji/vue-cesium/assets/29724308/b59d18f3-55bf-4f66-9a31-c70a92b05687)
就像这种一样，如果有交叉的话，会有drawTip，不允许用户操作等等

**Comment by zouyaoji** [comment]
也是drawEvt事件里面去判断，drawEvt的传参你输出来看一下。

**Reply by 1194964459** [reply]
若有交叉，如何动态更新对应的 drawTip？并且在界面上展示最新的drawTips呢？

**Reply by 1194964459** [reply]
可以更改某条线的颜色为某种单独的颜色吗？区别于整个polygon线条的颜色

**Reply by zouyaoji** [reply]
可以改哈，用ref拿到renderDatas.

https://github.com/zouyaoji/vue-cesium/discussions/206
